elections/election-2022.md

```diff
@@ -0,0 +1,95 @@
+# Project nomination for BSC representatves 2022
+
+In accordance with the [Elections Schedule](./schedule.md) and the
+[eBPF Foundation Charter][charter], this page holds a public record of the
+steps undertaken to nominate members for the BSC in 2022.
+
+The following members' terms will expire in 2022. The Section 4.b column
+indicates which projects must be considered to draw nominations from.
+
+| Name | Section 4.b |
+| ---- | ----------- |
+| TBD  |             |
+
+## Nominations (ETA August)
+
+The following nominees are under consideration for election to the BSC.
+
+### Kernel Candidates
+
+_[eBPF Foundation Charter][charter] §4.b.i Two representatives (the “Kernel
+Representatives”) from the group of eBPF Linux kernel maintainers (as specified
+in the maintainer file)._
+
+* TBD
+
+### Runtime Candidates
+
+_[eBPF Foundation Charter][charter] §4.b.ii One representative (each a “eBPF
+Runtime Representative”) for each additional open-source eBPF runtime
+implementation._
+
+* eBPF Runtime for Windows
+
+### Additional Project Candidates
+
+_[eBPF Foundation Charter][charter] §4.b.iii Up to two representatives
+(“Additional Project Representatives”) from additional open source projects
+that are important to the eBPF ecosystem, as determined by the BSC._
+
+* Aya (Rust Library)
+* Bumblebee
+* eCapture
+* hBPF
+* Hubble
+* Inspektor Gadget
+* KubeArmor
+* Kubectl Trace
+* L3AF
+* Libbpfgo
+* LibXDP
+* Parca
+* Pixie
+* Ply
+* Tracee
+
+### Maintainer Project Candidates
+
+_[eBPF Foundation Charter][charter] §4.b.iv Up to three representatives with
+maintainer status (“Maintainer Representatives”) of major eBPF Foundation
```

Comment:
Channelling Dave Thaler: none of these are eBPF Foundation Projects?

Reply:
This proposal is assuming the text refers to major landscape projects. The proposed wording amendments seem consistent (they leave this alone, rather than changing this to "Technical Project") ref. https://github.com/ebpffoundation/ebpf.io/pull/176

Reply:
^ Discussed during 2022-06-01, we need to clarify the charter in this regard first in order to clarify eligibility for this category.

Reply:
My opinion: it should be eBPF Foundation (not landscape) projects, but I think we should delay the new elections until after enough projects have gone through the process.

Reply:
The board has now approved the charter amendment proposed in https://github.com/ebpf-io/ebpf.io-website/pull/175/files